src/components/views/dialogs/SetEmailDialog.js

```diff
@@ -41,6 +41,7 @@ export default React.createClass({
     },
 
     componentDidMount: function() {
+        this.refs.emailInputField.focus();
```

Comment:
I think this would be better off done in the `EditableText` itself, given it already uses a `ref`.

Reply:
Most likelly I am missunderstanding your suggestion. I don't like automagically focusing each EditableText component as it renders as for example in the settings we do have multiple instances of that. If more instances tried to set the focus on render I can't imagine what might happen. Wouldn't it lead to unpredictable focus behaviour?

Reply:
You could easily pass an `autoFocus` prop to `EditableText` that will do the focussing. By default it could be `false`.